pages/dkp/konvoy/2.2/image-builder/override-files/default-or-files/index.md

```diff
@@ -10,9 +10,49 @@ menuWeight: 0
 
 Konvoy comes with default override files:
 
-- [FIPS](../../../fips/) override file.
-- [Nvidia](../../../choose-infrastructure/aws/gpu/) override file.
-- Offline override file.
-- Offline for FIPS override file.
+- FIPS override:
+  ```yaml
+  ---
+  k8s_image_registry: docker.io/mesosphere
 
-You can find these override files in the [Konvoy Image Builder repo](https://github.com/mesosphere/konvoy-image-builder/tree/main/overrides).
+  fips:
+  enabled: true
```

Comment:
Syntax broken here as opposed to the live files: https://github.com/mesosphere/konvoy-image-builder/blob/main/overrides/fips.yaml